src/node_modules/app/components/elementary/ObjectEditor.svelte

```diff
@@ -0,0 +1,95 @@
+<script context='module'>
+    let datasetTypings;
+</script>
+
+<script>
+    import { createEventDispatcher } from 'svelte';
+    import { getCompletions } from 'app/tsservices';
+	import { request } from 'app/net';
+
+	import ESField from './ElasticSearchField.svelte';
+	
+	const dispatch = createEventDispatcher();
+
+	export let id = null;
+	export let dataType;
+	export let required = false;
+	export let value = null;
+    export let defaultValue = null;
```

Comment:
there's a mix of tabs and spaces, we should use tabs only, here and elsewhere in the PR if the case

Reply:
I had linting turned off temporarily and this was a new file. I'll lint before I merge.